spec/index.html

```diff
@@ -615,6 +620,10 @@ <h3>RDF Terms</h3>
       <a>blank nodes</a>, and <a>triple terms</a> are collectively known as
       <span id="dfn-rdf-terms"><!-- obsolete term--></span><dfn data-lt="rdf term">RDF terms</dfn>.</p>
 
+    <p>
+      [=IRIs=], [=literals=] and [=blank nodes=] are said to be <dfn data-lt-no-plural>atomic</dfn> [=RDF terms=].
```

Comment:
Wrt. "atomic", see discussion in https://github.com/w3c/rdf-concepts/issues/172. (I'm all for the rest of this PR.)